components/helper/IndividualPageHead.tsx

```diff
@@ -10,7 +10,7 @@ export default function IndividualPageHead({ title, metaContent }: Props) {
   return (
     <Head>
       <title>{title}</title>
-      <meta name="description" content={metaContent} />
+      <meta name="description" content={metaContent} key="description" />
```

Comment:
On the line above, should `<title>` be `<meta title="">` ?

Reply:
Im not sure if we really need to change the `title` one.

If I understood the docs right, the `key` property is just necessary when we have meta tags that are used multiple times, so its to not duplicate, but to overwrite. 

And I didnt find any `title` tag used in `_document.tsx`. What do you think?

Reply:
@LeoFuna Hm I'm not sure honestly. Looking at the documentation you linked, the first example shows just the `<title>` but the second example also has a meta tag just below it. I'll leave it to you to choose, and we can always readjust if needed.

Reply:
Ok so I will try without changing the `title` just to not change something that might not be the problem.
And if that dont fix what we want, we can readjust as you said.